model/rql-parser/pom.xml

```diff
@@ -35,7 +35,13 @@
 
         <dependency>
             <groupId>com.typesafe.akka</groupId>
-            <artifactId>akka-http_${scala.version}</artifactId>
+            <artifactId>akka-parsing_${scala.version}</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.typesafe.akka</groupId>
+                    <artifactId>akka-actor_${scala.version}</artifactId>
+                </exclusion>
```

Comment:
Isn't akka-actor a "provided" dependency for akka-parsing?
Looks that way here: https://mvnrepository.com/artifact/com.typesafe.akka/akka-parsing_2.12/10.1.8

Reply:
True; deleting the exclusion does not introduce akka-actor as dependency. Will amend.